app/src/main/java/com/duckduckgo/app/autocomplete/api/AutoComplete.kt

```diff
@@ -134,7 +134,7 @@ class AutoCompleteApi @Inject constructor(
                 rankedBookmark.score += 50
             }
 
-        } else if (bookmark.url.toUri().toStringDropScheme().startsWith(tokens.first().trimEnd { it == '/' }, ignoreCase = true)) {
+        } else if (bookmark.url.toUri().toStringDropScheme().removePrefix("www.").startsWith(tokens.first().trimEnd { it == '/' }, ignoreCase = true)) {
```

Comment:
This logic is getting more difficult to read. Would you mind extracting it to a function that explains what it does for better readability?

Reply:
Done